src/gt4py/storage/cartesian/interface.py

```diff
@@ -12,11 +12,15 @@
 #
```

Comment:
It is kind of in the middle. Formally it's an API change because the renamed modules didn't start with an underscore and then they were assumed to be public interface, but on the other hand, the reexports at the storage level are the preffrfed way to use this symbols and this still works ...